classes/tables/entries_table.php

```diff
@@ -36,15 +36,22 @@ class entries_table extends table_sql {
      * @param $cm mixed The course module.
      * @param $userid string The id for the user.
      * @param $output renderer_base The output renderer to use.
+     * @param $actions boolean If true, the actions column is displayed.
```

Comment:
I'd probably rename this to $download because there is likely to be a bunch of fields that we add or remove depends on whether it is being downloaded. For instance any fields that we end up deciding not to include in the table should still be in the download